recipes/openjpeg/all/conanfile.py

```diff
@@ -105,27 +103,31 @@ def _module_subfolder(self):
         return os.path.join("lib", "cmake")
 
     @property
-    def _module_file(self):
-        return "conan-official-{}-targets.cmake".format(self.name)
+    def _module_file_rel_path(self):
+        return os.path.join(self._module_subfolder,
+                            "conan-official-{}-targets.cmake".format(self.name))
 
     @property
     def _openjpeg_subdir(self):
         openjpeg_version = tools.Version(self.version)
         return "openjpeg-{}.{}".format(openjpeg_version.major, openjpeg_version.minor)
 
     def package_info(self):
+        self.cpp_info.set_property("cmake_file_name", "OpenJPEG")
+        self.cpp_info.set_property("cmake_target_name", "openjp2")
+        self.cpp_info.set_property("pkg_config_name", "libopenjp2")
```

Comment:
It breaks `ffmpeg` build because it can't find `libopenjp2`, with conan 1.44.0 at least. Indeed, now pkg_config generator creates `openjpeg.pc`
Actually `pkg_config` generator doesn't seem to listen `set_property()` anymore, so we need to still maintain `.names["pkg_config"]` until conan v2. It's really unfortunate.

@czoido was it intended? I thought that only `cmake_find_package*` stopped to listen `set_property()`.

Reply:
Today or tomorrow the latest we will release `1.44.1 `and `1.43.3` fixing this bug, and `pkg_config` will listen to `set_property` for the root `cpp_info` as well, sorry for that.